scripts/o3de/example_project_export_scripts/newspaper_delivery_project_export_workflow.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
In favor of the python based solution for the short term: This is less human readable than Python.

I think a json based workflow system works best with a UI of some kind to manage the file